### ISSUE TITLE
Focus Mode : Scratch (instead of scratch layer) 

### ISSUE THREAD

**Issue by PHHENS** [paperwm/PaperWM#869]
Thanks for already done work :-)
Would like to have a normal placement and sizing window mode into some workspaces (like a scratch focus mode). 
Could be  usefull with some apps (like sticky notes).

Thanks for understanding this need.

**Comment by jtaala** [comment]
> Thanks for understanding this need.

I'm not understanding this - do you mean a scratch mode, but just per workspace?

**Comment by PHHENS** [comment]
Yes, just a tradionnal (no tiling) floating layout per workspace.
So we could freely size and position windows in those selected 
workspaces.
Typical usage : on workspace with sticky notes windows freely 
positionned and sized.

Thanks for understanding.

**Comment by Thesola10** [comment]
So wait, this would amount to pretty much disabling PaperWM on a per-workspace basis? Or sending any windows opened while a certain workspace is focused to the scratch layer, while leaving the possibility to tile them?

The latter sounds like a good improvement in general as it could help quickly fix situations where windows are tiled when they shouldn't be (e.g. LibreOffice Impress)
